src/node_watchdog.cc

```diff
@@ -229,9 +230,9 @@ void TraceSigintWatchdog::HandleInterrupt() {
 
 #ifdef __POSIX__
 void* SigintWatchdogHelper::RunSigintWatchdog(void* arg) {
+  uv_thread_setname("SigintWatchdogWorker");
```

Comment:
I think the term "Worker" is not accurate for the watchdog threads. The workdog threads are usually idle until being triggered.

```suggestion
  uv_thread_setname("SigintWatchdog");
```